src/main/kotlin/com/hibob/academy/filters/AuthenticationFilter.kt

```diff
@@ -0,0 +1,49 @@
+package com.hibob.academy.filters
+
+import com.hibob.academy.service.SessionService
+import io.jsonwebtoken.Claims
+import io.jsonwebtoken.Jws
+import io.jsonwebtoken.Jwts
+import jakarta.ws.rs.container.ContainerRequestContext
+import jakarta.ws.rs.container.ContainerRequestFilter
+import jakarta.ws.rs.core.Cookie
+import jakarta.ws.rs.core.Response
+import jakarta.ws.rs.ext.Provider
+import org.springframework.stereotype.Component
+
+const val PATH_TO_SKIP = "api/auth/login"
+
+const val AUTH_COOKIE_NAME = "Authorization"
```

Comment:
we usually put consts in a companion object. look in the ExampleDao for an example

Reply:
got it, thanks